src/compiler/generate_cargo_toml_files.ts

```diff
@@ -28,7 +28,9 @@ export function generateWorkspaceCargoToml(optLevel: OptLevel): Toml {
 
         [workspace]
```

Comment:
Do we even need this? Could we delete all of worspace? we seemed to be living quite happily with canister_methods this whole time

Reply:
No it needs the workspace at least, so might as well put it all in?

Reply:
sounds good to me